server/src/main/java/io/spine/server/command/AbstractCommander.java

```diff
@@ -107,11 +108,11 @@ public void dispatch(CommandEnvelope command) {
 
     @Override
     public void dispatchEvent(EventEnvelope event) {
-        CommandReactionMethod method = thisClass.commanderOn(event.messageClass());
-        DispatchOutcomeHandler
-                .from(method.invoke(this, event))
+        Optional<CommandReactionMethod> method = thisClass.commanderOn(event);
```

Comment:
The question here is why the runtime execution has come to this point while the filtering rules may have not matched. I would think it should have been figured out at the dispatching stage.

It's not too scary with the commanders, but with `Entity` instances it's much worse, as we are going to load them from their storage.

Let's of course keep returning `Optional` here, but in case it's `empty`, there should be some consequences. Not silence.